src/CodeGeneration/ClassDetailsResolver/AppClassDetailsResolver.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Dealroadshow\Bundle\K8SBundle\CodeGeneration\ClassDetailsResolver;
+
+use Dealroadshow\Bundle\K8SBundle\CodeGeneration\ClassDetails;
+use Dealroadshow\Bundle\K8SBundle\Util\Str;
+use Dealroadshow\K8S\Framework\Project\ProjectInterface;
+use ReflectionObject;
+
+class AppClassDetailsResolver
+{
+    private const SUFFIX_APP = 'App';
+    private const APPS_DIR_NAME = 'Apps';
+
+    private string $namespacePrefix;
+
+    public function __construct(string $namespacePrefix)
+    {
+        $this->namespacePrefix = trim($namespacePrefix, '\\');
+    }
+
+    public function getClassDetails(ProjectInterface $project, string $appName): ClassDetails
+    {
+        $className = $this->className($appName);
+        $namespace = $this->getNamespace($project, $appName);
```

Comment:
I think you should name all methods with prefix `get`, or all methods without it. Now you have method `className`, but `getNamespace`